Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:latest
+FROM python:3.10
```

Comment:
I understand the crfsuite is putting a constrain on which python version it would work with. This might be a long term maintenance project where 3.10 would be too old and needs to be updated.

Reply:
I think for now it will be okay to keep it at 3.10 but yeah long term maintenance would be a little more tricky with explicitly passing a python version image

Reply:
created an issue  #591